.github/workflows/create-tag-with-release-pr.yml

```
@@ -0,0 +1,84 @@
+name: Create release tag on release PR merge
+on:
+  pull_request:
+    types:
+     - closed
+
+jobs:
+  create-release-tag:
+    runs-on: ubuntu-latest
```

It's inconsistent with the rest of the workflow, this job uses latest whereas the other job uses a constant major version:
```suggestion
    runs-on: ubuntu-24.04
```